### PR TITLE
TEP-0111 - Propagating Workspaces.

### PR DESCRIPTION
**Pull request by chitrangpatel**
`Tekton Pipelines` resources can be quite verbose because of explicitly propagating `Workspaces`. This proposal aims to propagate `Workspaces` to improve usability of `Tekton Pipelines`.

### CODE REVIEW

File: teps/0111-propagating-workspaces.md [hunk 1]
```diff
@@ -0,0 +1,615 @@
+---
+status: proposal
+title: Propagating Workspaces
+creation-date: '2022-06-03'
+last-updated: '2022-06-03'
+authors:
+- '@chitrangpatel'
+- '@jerop'
+---
```
Comment (jerop) [inline teps/0111-propagating-workspaces.md:9]:
```suggestion
see-also:
- TEP-0107
- TEP-0108
---
```

File: teps/0111-propagating-workspaces.md [hunk 2]
```diff
@@ -0,0 +1,618 @@
...
+status: proposal
```
Comment (jerop) [inline teps/0111-propagating-workspaces.md:2]:
```suggestion
status: implementable
```

@chitrangpatel here's the status options: https://github.com/tektoncd/community/blob/main/teps/0001-tekton-enhancement-proposal-process.md#tep-workflow

File: teps/README.md [hunk 3]
```diff
@@ -250,3 +250,4 @@ This is the complete list of Tekton teps:
 |[TEP-0107](0107-propagating-parameters.md) | Propagating Parameters | implemented | 2022-05-26 |
 |[TEP-0108](0108-mapping-workspaces.md) | Mapping Workspaces | implemented | 2022-05-26 |
 |[TEP-0110](0110-decouple-catalog-organization-and-reference.md) | Decouple Catalog Organization and Resource Reference | implementable | 2022-06-06 |
+|[TEP-0111](0111-propagating-workspaces.md) | Propagating Workspaces | proposal | 2022-06-03 |
```
Comment (jerop) [inline teps/README.md:253]:
```suggestion
|[TEP-0111](0111-propagating-workspaces.md) | Propagating Workspaces | implementable | 2022-06-03 |
```

File: teps/0111-propagating-workspaces.md (hunk 1 above)
Comment (chitrangpatel) [reply teps/0111-propagating-workspaces.md:12]: done

File: teps/README.md (hunk 3 above)
Comment (jerop) [reply teps/README.md:253]: this is not yet fixed, that's why linting is failing

Comment (chitrangpatel) [reply teps/README.md:253]: Oh it was in the README. Thanks

**Review by dibyom — APPROVED** [review]

**Review by vdemeester — APPROVED** [review]

**Pull request merged**